modules/olm/crds.tf

```diff
@@ -0,0 +1,17 @@
+module "yaml" {
+  source = "../../../terraform-null-helpers/modules/yaml"
```

Comment:
this module source pointing local not existing module, please have it deployed into terraform registry and use remote registry source